Source/ThirdParty/libwebrtc/Configurations/Base.xcconfig

```diff
@@ -72,7 +72,11 @@ GCC_WARN_UNUSED_FUNCTION = YES;
 GCC_WARN_UNUSED_VARIABLE = YES;
 CLANG_WARN_SUSPICIOUS_MOVE = YES;
 PREBINDING = NO;
-WARNING_CFLAGS = -Wexit-time-destructors -Wformat=2 -Wglobal-constructors -Wthread-safety;
+WARNING_CFLAGS = $(WK_COMMON_WARNING_CFLAGS) $(WK_FIXME_WARNING_CFLAGS) -Wexit-time-destructors -Wglobal-constructors -Wthread-safety;
+
+// Remove WK_FIXME_WARNING_CFLAGS once all warnings are fixed.
+// -Wno-unknown-warning-option added for -Wno-unused-but-set-parameter.
+WK_FIXME_WARNING_CFLAGS = -Wno-conditional-uninitialized -Wno-deprecated-copy -Wno-missing-field-initializers -Wno-sign-compare -Wno-undef -Wno-unknown-warning-option -Wno-unused-but-set-parameter -Wno-unused-but-set-variable -Wno-unused-parameter;
```

Comment:
I have a feeling these will never be fixed.

Reply:
> I have a feeling these will never be fixed.

1. I feel it's better to document them than to leave them as unknowns, even if they weren't fixed.
2. I literally plan to fix some (not all) of these after this patch lands as many have fewer than 10 results, but this patch was already getting big, so I didn't want to wrap up all those changes into this commit.